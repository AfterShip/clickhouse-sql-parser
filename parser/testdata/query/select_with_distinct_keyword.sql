SELECT DISTINCT record_id FROM records 