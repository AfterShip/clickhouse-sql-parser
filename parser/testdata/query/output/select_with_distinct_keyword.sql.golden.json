[
  {
    "SelectPos": 0,
    "StatementEnd": 38,
    "With": null,
    "Top": null,
    "HasDistinct": true,
    "SelectItems": [
      {
        "Expr": {
          "Name": "record_id",
          "QuoteType": 1,
          "NamePos": 16,
          "NameEnd": 25
        },
        "Modifiers": [],
        "Alias": null
      }
    ],
    "From": {
      "FromPos": 26,
      "Expr": {
        "Table": {
          "TablePos": 31,
          "TableEnd": 38,
          "Alias": null,
          "Expr": {
            "Database": null,
            "Table": {
              "Name": "records",
              "QuoteType": 1,
              "NamePos": 31,
              "NameEnd": 38
            }
          },
          "HasFinal": false
        },
        "StatementEnd": 38,
        "SampleRatio": null,
        "HasFinal": false
      }
    },
    "ArrayJoin": null,
    "Window": null,
    "Prewhere": null,
    "Where": null,
    "GroupBy": null,
    "WithTotal": false,
    "Having": null,
    "OrderBy": null,
    "LimitBy": null,
    "Limit": null,
    "Settings": null,
    "Format": null,
    "UnionAll": null,
    "UnionDistinct": null,
    "Except": null
  }
]